Packs/SimpleSFTP/Integrations/SimpleSFTP/SimpleSFTP.py

```diff
@@ -0,0 +1,70 @@
+import paramiko
+import traceback
+
+
+def main():
+    HOST = demisto.params()["host"]
+    USERNAME = demisto.params()['authentication']['identifier']
+    PASSWORD = demisto.params()['authentication']['password']
+    PORT = int(demisto.params()["port"])
+
+    if demisto.command() == "test-module":
+        try:
+            client = paramiko.Transport(HOST, PORT)
+            client.connect(username=USERNAME, password=PASSWORD)
+            sftp = paramiko.SFTPClient.from_transport(client)
+            sftp.close()
+            demisto.results("ok")
+        except Exception as ex:
+            demisto.error(traceback.format_exc())  # print the traceback
+            return_error(f'Failed to connect Error: {str(ex)}')
+
+    if demisto.command() == "sftp-listdir":
+        try:
+            client = paramiko.Transport(HOST, PORT)
+            client.connect(username=USERNAME, password=PASSWORD)
+            sftp = paramiko.SFTPClient.from_transport(client)
+            directory = demisto.args()["directory"]
+            res = sftp.listdir(path=directory)
+            entry = {
+                'Type': entryTypes['note'],
+                'ContentsFormat': formats['text'],
+                'Contents': res,
+                'ReadableContentsFormat': formats['text'],
+                'HumanReadable': res,
+                'EntryContext': {"SFTP.ListDir": res}
+            }
+            demisto.results(entry)
+            sftp.close()
+        except Exception as ex:
+            demisto.error(traceback.format_exc())  # print the traceback
+            return_error(f'Error occurred - Error: {str(ex)}')
+
+    elif demisto.command() == "sftp-copyfrom":
+        try:
+            client = paramiko.Transport(HOST, PORT)
+            client.connect(username=USERNAME, password=PASSWORD)
+            sftp = paramiko.SFTPClient.from_transport(client)
+            filePath = demisto.args()["filePath"]
+            res = sftp.get(filePath, "/tmp/" + filePath[filePath.rindex("/") + 1:])
+            sftp.close()
+            with open("/tmp/" + filePath[filePath.rindex("/") + 1:], "r") as f:
+                data = f.read()
+                if demisto.args()["returnFile"] == "True":
+                    demisto.results(fileResult(filename=filePath[filePath.rindex("/") + 1:], data=data))
+            entry = {
+                'Type': entryTypes['note'],
```

Comment:
You don't want to return a file instead of plain text (I mean take in mind what if the file will be huge).
Please see [here](https://xsoar.pan.dev/docs/integrations/context-and-outputs#return-file)